Actions.js

```diff
@@ -21,64 +21,86 @@ router.get('/:id', (req, res) => {
     db.get( id )
 
      .then(action => {
+       if (!id) 
+       res.status(404).json({
+         message: "Could not find action with this ID."
```

Comment:
good job with adding error handling so it's more descriptive to what is happening on a successful query of a resource that does not exist.